dashboard/app/views/scripts/show.html.haml

```diff
@@ -6,7 +6,8 @@
 - scriptOverviewData = {scriptData: @script_data, plcBreadcrumb: @plc_breadcrumb}
 
 - content_for(:head) do
-  %script{ src: webpack_asset_path('js/scripts/show.js'), data: {scriptoverview: scriptOverviewData.to_json }}
+  %script{src: webpack_asset_path('js/scripts/show.js'),
+          data: {scriptoverview: scriptOverviewData.to_json, parental_permission_banner: current_user && parental_permission_banner_data(current_user, request).to_json}}
```

Comment:
Technically, this sets the field to `"null"` when there is no banner data... and that does work since it is parsed in `show.js`. The logic in `show.js` seems to support this dataset field being actual `null` or undefined. Could we have the data field not exist in that case?